tests/run_tests.py

```diff
@@ -200,6 +204,11 @@ def _parse_command_line():
     :returns: The options and the name of the unit test specified on the command line, if any.
     """
     parser = OptionParser()
+    # TODO: ask aout the 'action' params below
```

Comment:
You can get rid of that option. we'll always clean the files for now. No need to opt in.